k8s/analytics/values-stage.yaml

```diff
@@ -1,35 +1,50 @@
-# values for airqo-analytics-api.
-env:
-  deployment_name: airqo-stage-analytics-api
-  service_name: airqo-stage-analytics-api-svc
-  labels: sta-analytics-api
-  secret: env-analytics-staging
-
 replicaCount: 2
 
 image:
-  repository: us.gcr.io/airqo-250220/airqo-stage-analytics-api
+  repository: gcr.io/airqo-250220/airqo-stage-analytics-api
```

Comment:
Staging values files

Reply:
I am just giving some high level analysis.....just curious. 
Isn't Helm supposed to reduce the prevent us from having different files for the different environments? 
Please help shed more light on this. Thanks!

Reply:
Yes @Baalmart 
It does exactly that. Those are the template files.
The values files have specific values for the different files which do not repeat.
For example, you'll find the the namespace values in the two files are totally different i.e in one file staging, in the other production